core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/worker/DataProcessor.scala

```diff
@@ -93,12 +93,47 @@ class DataProcessor(
     }
   }
 
+  private[this] def processInputState(state: State, port: Int): Unit = {
+    try {
+      val outputState = executor.processState(state, port)
+      if (outputState.isDefined) {
+        outputManager.emitMarker(outputState.get)
+      }
+    } catch safely {
+      case e =>
+        // forward input tuple to the user and pause DP thread
```

Comment:
please avoid wrong comments. here there is no tuple right?

Reply:
Fixed.